package.json

```diff
@@ -9,7 +9,7 @@
     "build": "NODE_ENV=production tsc",
     "postbuild": "copyfiles -f -V src/views/* dist/app/views",
     "lint": "tslint -p .",
-    "test": "jest -c src/test/jest.config.js"
+    "test": "NODE_ENV=test jest --runInBand --coverage"
```

Comment:
A small comment that's up for debate:
You could possibly add a `test:coverage` script to get a coverage report while keeping the `test` script as a simple test run, the simple test script will run a little faster without coverage, and even if it is only a small difference, it still might be nice to have the option to run with or without coverage.
i.e.
```
"test": "NODE_ENV=test jest --runInBand",
"test:coverage": "NODE_ENV=test jest --runInBand --coverage"
```